src/gallery.ts

```diff
@@ -212,56 +207,61 @@ export class Gallery {
     ).toString();
 
     const root = PATHS.mobjImgs.fsPath;
-    var newVersion = localVersion;
-    Axios.get(GITHUB_ENTRY_FILE).then(({ data }) => {
-      if (!forceDownload) {
-        const version = data.match(VERSION_RE);
-        if (!version) {
-          vscode.window.showErrorMessage(
-            "Version descriptor in remote location missing. Please try again later."
-          );
-          return;
-        }
+    var {data} = await axios.get(GITHUB_ENTRY_FILE);
+    data = yaml.load(data) as any; // Parse yaml
+    
+    let newVersion = data["user_content_version"];
+    
+    // Related to https://github.com/kolibril13/mobject-gallery/issues/3
+    const galleryParameters = https://github.com/kolibril13/mobject-gallery/issues/3
+      data["gallery_parameters_path"] ||
+      data["gallary_parameters_path"];
```

Comment:
```suggestion
    const galleryParameters = data["gallery_parameters_path"];
```
Issue seems to be fixed before the merge!